src/AppInstallerCLICore/Commands/COMInstallCommand.cpp

```diff
@@ -27,8 +27,7 @@ namespace AppInstaller::CLI
     void COMInstallCommand::ExecuteInternal(Context& context) const
     {
         context <<
-            Workflow::GetInstallerHash <<
-            Workflow::VerifyInstallerHash <<
+            Workflow::ReverifyInstallerHash <<
```

Comment:
nit: This seems overly aggressive on combining tasks.

Reply:
We'll leave it as is for bug fix now